package.json

```diff
@@ -26,6 +27,7 @@
     "angular-mocks": "1.5.8",
     "angular-sanitize": "1.5.8",
     "grunt": "^0.4.5",
+    "grunt-cli": "^1.2.0",
```

Comment:
Isn't the CLI meant to be installed as a global? Not so sure this is a great change.